### PR TITLE
add an issue template and rename feature request to proposal

### PR DESCRIPTION
**Pull request by hormes**
Signed-off-by: Fansong Zeng <fanster.z@gmail.com>

### Ⅰ. Describe what this PR does
add an issue template for user support and rename feature request to proposal,  we encourage questions and solutions.

### CODE REVIEW

**Comment by hormes** [comment]
please review this pr, thanks @jasonliu747

File: .github/ISSUE_TEMPLATE/proposal.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Proposal
+about: Suggest an idea for this project
+title: "[proposal]"
+labels: kind/proposal
```
Comment (jasonliu747) [inline .github/ISSUE_TEMPLATE/proposal.md:5]: do we support this label right now?

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**